README.md

```diff
@@ -97,11 +110,12 @@ services.AddStatsD(
     {
         var options = provider.GetRequiredService<MyOptions>().StatsD;
 
-        return new StatsDConfiguration()
+        return new StatsDConfiguration
         {
             Host = options.HostName,
             Port = options.Port,
             Prefix = options.Prefix,
+            SocketProtocol = SocketProtocol.Ip,
```

Comment:
Suggest just changing this to `options.Protocol`, as otherwise the docs could be interpreted as saying _"you must set this to this in this scenario"_ as there is no other commentary here. Or, just omit it entirely as it seems more the sort of option that's for advanced usage that "normal config".

Reply:
AFIK, you _must_ set this in the AWS lambda scenario.

Reply:
My mistake, I thought this was just the generic IoC example.